pkg/assembler/backends/ent/backend/vulnerability.go

```diff
@@ -70,15 +64,21 @@ func (b *EntBackend) Vulnerabilities(ctx context.Context, filter *model.Vulnerab
 }
 
 func getVulnerabilities(ctx context.Context, client *ent.Client, filter model.VulnerabilitySpec) (ent.VulnerabilityTypes, error) {
+	if filter.ID != nil {
+		_, err := strconv.Atoi(*filter.ID)
+		if err != nil {
+			return nil, err
+		}
+	}
 	var where = []predicate.VulnerabilityType{
-		optionalPredicate(filter.ID, IDEQ),
```

Comment:
Why has this condition been removed?
Filtering by ID should work.

Reply:
Restored